tests/jobs/test_scheduler_job.py

```diff
@@ -3687,7 +3687,6 @@ def complete_one_dagrun():
         for _ in range(5):
             self.scheduler_job._do_scheduling(session)
             complete_one_dagrun()
-            model: DagModel = session.query(DagModel).get(dag.dag_id)
```

Comment:
Didn't need this one?

Reply:
I guess not, this `model` not use later in the code, we just make a query to DB and that all.

This part unchanged since it added in the end if 2021: https://github.com/apache/airflow/pull/19528